### PR TITLE
MAT-6631: Combine Population Description into One Value For Exports

### PR DESCRIPTION
**Pull request by ethankaplan**
…requirements

## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6631](https://jira.cms.gov/browse/MAT-6631)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java [hunk 1]
```diff
@@ -116,60 +115,75 @@ private void setMeasurementPeriodForQdm(HumanReadableMeasureInformationModel mod
...
+            .transmissionFormat(measure.getMeasureMetaData().getTransmissionFormat())
+            .definition(
+                HumanReadableUtil.escapeHtmlString(measure.getMeasureMetaData().getDefinition()))
+            .references(HumanReadableUtil.buildReferences(measure.getMeasureMetaData()))
+            .build();
+    generatePopulatiosn(measure, modelTemp);
+    return modelTemp;
+  }
+
+  private void generatePopulatiosn(
```
Comment (nmorasb) [inline src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java]: please fix typo in method name

File: src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java [hunk 2]
```diff
@@ -124,4 +128,47 @@ public static String getPopulationDescription(Measure measure, String population
     }
     return sb.toString();
   }
+
+  public static String getMeasureObservation(Measure measure) {
```
Comment (nmorasb) [inline src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java]: can we refactor / rename this to `getMeasureObservationDescriptions` to better indicate what we're fetching are the descriptions, not the whole measure observation object

File: src/test/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtilTest.java
Comment (nmorasb) [inline src/test/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtilTest.java:1]: I think we might want some unit tests for the new methods that were added..some methods were moved over from the HumanReadableService, so coverage of those _could_ be considered tech debt. At the very least, unit tests should be added to cover the new method `getMeasureObservationDescriptions` (after name is refactored)

Comment (ethankaplan) [reply src/test/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtilTest.java:1]: Unit: tested

**Review by nmorasb — APPROVED** [review]
